Examples/MAX32665/SPI_3Wire/README.md

```diff
@@ -0,0 +1,46 @@
+## Description
+
+This example sends data between two SPI peripherals operating in 3-wire SPI mode.
+
+## Software
+
+### Project Usage
+
+Universal instructions on building, flashing, and debugging this project can be found in the **[MSDK User Guide](https://analog-devices-msdk.github.io/msdk/USERGUIDE/)**.
+
+### Project-Specific Build Notes
+
+(None - this project builds as a standard example)
```

Comment:
Is the project supported on the EVKIT?  If not, add a note here.

Reply:
I added the note.